code/modules/hydroponics/grown/towercap.dm

```diff
@@ -19,7 +19,7 @@
 	icon_dead = "towercap-dead"
 	genes = list(/datum/plant_gene/trait/plant_type/fungal_metabolism)
 	mutatelist = list(/obj/item/seeds/tower/steel)
-	reagents_add = list("carbon" = 0.2)
+	reagents_add = list("plantmatter" = 0.225)
```

Comment:
the `.225` looks really strange, i cant find any other similar levels of reagents, aside from sugar thats `.25`. others may need to take a look at  this. 

Aside from that, very good pr

Reply:
yeah i dont understand why the % is double of whatever the reagent list is so if i wanted 45% plantmatter id have to put 0.225 since that doubles to 0.45 in the plant manipulator thing. the first time i tried editing it i put 0.45 and it ended up as 90% plant matter